demisto_sdk/commands/common/tests/release_notes_test.py

```diff
@@ -393,3 +394,78 @@ def test_has_release_notes_been_filled_out(release_notes, filled_expected_result
     mocker.patch.object(UpdateRN, 'get_master_version', return_value='0.0.0')
     validator = get_validator(release_notes, MODIFIED_FILES)
     assert validator.has_release_notes_been_filled_out() == filled_expected_result
+
+
+TEST_RELEASE_NOTES_TEST_BANK_3 = [
+    ('#### Integrations\n##### Integration name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.',
+     {'display': 'Integration name', 'script': {'dockerimage': 'demisto/python3:3.9.5.21272'}}, True),
+    ('\n#### Scripts\n##### Script name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.21272*.',
+     {'name': 'Script name', 'script': {'dockerimage': 'demisto/python3:3.9.5.21272'}}, True),
+    ('\n#### Scripts\n##### Script name\n- Upgraded the Docker image to: *demisto/python3:3.9.5.2122*.',
+     {'name': 'Script name', 'script': {'dockerimage': 'demisto/python3:3.9.4.272'}}, False),
+    ('#### Integrations\n##### Integration name\n- Moved the Pack to Cortex XSOAR support instead of community support.',
+     {'display': 'Integration name', 'script': {'dockerimage': 'demisto/python3:3.9.5.21272'}}, True),
+]
+
+
+@pytest.mark.parametrize('release_notes_content, yaml_content, filled_expected_result', TEST_RELEASE_NOTES_TEST_BANK_3)
+def test_is_docker_image_same_as_yml(release_notes_content, yaml_content, filled_expected_result, pack: Pack):
+    """
+    Given
+    - Case 1: Validate for integrations, version match.
+    - Case 2: Validate for scripts, version match.
+    - Case 3: Valid when version doesnt match.
+    - Case 4: Release notes appear but docker not changed.
+
+    When
+    - Running validation on release notes.
+
+    Then
+    - Ensure validation correctly identifies valid release notes.
+    - Case 1: Should print nothing and return True.
+    - Case 2: Should print nothing and return True.
+    - Case 3: Should return the prompt "Release notes dockerimage version does not match yml dockerimage version." and
+            return False.
+    - Case 4: Should print nothing and return True.
```

Comment:
Nice testing!!
1. In your 2nd case you are saying you are running the test on a script but in the test itself you are creating an integration yml: `integration1 = pack.create_integration()`
2. In the 3rd case you say "Valid when version doesn't match." What do you mean?

```suggestion
    Given
    - Case 1: RN containing a docker update, integration YML containing a docker update, where docker image equal in both.
    - Case 2:  RN containing a docker update, script YML containing a docker update, where docker image equal in both.
    - Case 3: Valid when version doesn't match.
    - Case 4: Release notes without a docker update, YML without a docker update.

    When
    - Running validation on release notes.

    Then
    - Ensure validation correctly identifies valid release notes.
    - Case 1: Should print nothing and return True.
    - Case 2: Should print nothing and return True.
    - Case 3: Should return the prompt "Release notes dockerimage version does not match yml dockerimage version." and
            return False.
    - Case 4: Should print nothing and return True.
```